packages/blocks/src/lib/navigation-map/components/plan.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+import type { LngLat } from '$lib/maplibre/types';
+import { T } from '@threlte/core';
+import { Color, type ColorRepresentation } from 'three';
+
+export let plan: LngLat[];
+export let color: ColorRepresentation;
+
+// The typings for color are incorrect - they expect a number but need a THREE.Color.
+$: threeColor = new Color(color) as unknown as number;
```

Comment:
We fixed this upstream in `@types/three`. This should be resolved up updating everything. Up for me submitting a dependency update PR today so that we don't have to do this?

Reply:
That would be awesome, thanks!

Reply:
Merged! You should be able to clean this up now ✨

Reply:
Thank you!!

Reply:
Done!